### PR TITLE
Webpack instructions

### PR DESCRIPTION
**Pull request by wtilton**

### CODE REVIEW

File: doc/article/en-US/validation-basics.md [hunk 1]
```diff
@@ -34,6 +36,12 @@ If you're using the `aurelia-cli`, add the following configuration to your `aure
 
 If you're not sure where to put this, search your `aurelia.json` for *aurelia-templating-resources* and put it underneath.
 
+### Webpack
+
+If you're using webpack, go to your `webpack.config.vendor.js` file and insert 'aurelia-validation' in the entry.vendor section of the returned object.
```
[the remark this answers is not in the document]
Comment (wtilton) [reply doc/article/en-US/validation-basics.md]: Just trying to help the webpack people. Currently using the VS SPA template and have no idea how to get it going. I'm new to both though so I suppose use it or modify as needed.

**Comment by EisenbergEffect** [comment]
@jods4 Can you review whether this doc update is correct?

**Comment by jods4** [comment]
It's good additions but I feel like they lack some contexts.

In particular the first one. I understand its purpose and it's a good tip but not everyone has a `webpack.config.vendor.js` file. Not everyone is even using a vendor bundle for that matter.
As is, it's a bit of cargo cult as it tells you what to do, not why. 
I think it would be slightly betters if it explicitely said somewhere "if you're using a vendor bundle, `aurelia-validation` should be added to it. [You can do that by...]" or something to that effect.

The second one is good as it correctly shows how to bootstrap `aurelia-validation`. Again, maybe a bit more context like a larger piece of code would be nice, as not everyone has a `boot.ts` file. Maybe say "in your main configure function..." and add a bit more code like at least the object on which `.plugin` is called...

**Comment by jdanyow** [comment]
thanks @wtilton, @jods4, 

@wtilton would you be willing to make @jods4's suggested changes?

**Pull request closed without merging**